clio-client/src/test/scala/org/broadinstitute/clio/client/metadata/CramDelivererSpec.scala

```diff
@@ -51,10 +51,12 @@ class CramDelivererSpec extends FlatSpec with Matchers {
     )
   private val destination = URI.create("gs://the-destination/")
 
-  private val deliverer = new CramDeliverer
+  private val delivererWithMetrics = CramDeliverer(deliverSampleMetrics = true)
 
-  it should "generate ops to move the cram & crai, and write the cram md5" in {
-    val (delivered, ops) = deliverer.moveInto(metadata, destination)
+  private val delivererNoMetrics = CramDeliverer(deliverSampleMetrics = false)
+
+  it should "generate ops to move the cram & crai, write the cram md5 and copy metrics files" in {
+    val (delivered, ops) = delivererWithMetrics.moveInto(metadata, destination)
```

Comment:
Please add a check to verify that the metrics files are in the right place

Reply:
This is already done below, see the lines regarding `preAdapterSummaryMetricsPath`, `preAdapterDetailMetricsPath`, etc.

Reply:
github autocollapse strikes again.